_data/tools/hdm.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "hdm",
+    "display_name": "Hiera Data Manager",
+    "description": "Web interface for Hiera data",
+    "url": "https://github.com/betadots/hdm"
+}
```

Comment:
Can you please add the missing \n?

Reply:
Done, had to force push to verify sign commits